nix/worker.nix

```diff
@@ -50,11 +62,6 @@ in
         default = config.networking.hostName;
         description = "The buildbot worker name.";
       };
-      nixEvalJobs.package = lib.mkOption {
-        type = lib.types.package;
-        default = pkgs.callPackage ./nix-eval-jobs.nix { };
```

Comment:
Might be best to keep this option and change the default to `pkgs.nix-eval-jobs`. It will make it eaiser to test new nix-eval-jobs versions.

Reply:
Should be fine.